build/includes/sdk.mk

```diff
@@ -155,7 +156,10 @@ run-sdk-conformance-test-node:
 	$(MAKE) run-sdk-conformance-test SDK_FOLDER=node GRPC_PORT=9002 HTTP_PORT=9102
 
 run-sdk-conformance-test-go:
-	$(MAKE) run-sdk-conformance-test SDK_FOLDER=go   GRPC_PORT=9001 HTTP_PORT=9101
+	# run without feature flags
+	$(MAKE) run-sdk-conformance-test SDK_FOLDER=go GRPC_PORT=9001 HTTP_PORT=9101
+	# run with feature flags enabled
+	$(MAKE) run-sdk-conformance-no-build SDK_FOLDER=go GRPC_PORT=9001 HTTP_PORT=9101 FEATURE_GATES=PlayerTracking=true TESTS=ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch,reserve,getplayercapacity,setplayercapacity
```

Comment:
for `TESTS` should we define a variable somewhere with alpha conformance tests so that we don't have to repeat this block elsewhere?

Reply:
Nice. I'll create a "standard" one and a "Alpha" variable too.